test/e2e/deploy_reaper_test.go

```diff
@@ -36,31 +38,21 @@ var (
 var _ = Describe("Deploy Reaper with Cassandra backend", func() {
 	Context("When a Cassandra cluster is deployed", func() {
 		Specify("Reaper is deployed", func() {
+			// Speed up cassandradatacenter_controller status checks
+			os.Setenv("REQUEUE_DELAY_STATUS_CHECK", "1m")
```

Comment:
Not sure if you aware, but we override the env vars, inlcuding `REQUEUE_DELAY_STATUS_CHECK`, with a kustomize patch.

Take a look at the output of `kustomize build test/config/deploy_reaper_test` to see.

This is done in `test/config/deploy_reaper_test/reaper-operator-patch.yaml`.